ci/cron/daily-compat.yml

```diff
@@ -34,7 +34,7 @@ jobs:
 
   - job: compatibility
     dependsOn: compatibility_ts_libs
-    timeoutInMinutes: 240
+    timeoutInMinutes: 360
```

Comment:
I’ve bumped the timeout since we’ve already seen timeouts and this makes things even slower. Thinking about ways to make this faster, maybe it would make sense to skip the first few steps and directly jump to the first messaging feature with all deps. That probably cuts time in half at least. That would mean that we will have to keep both tests but that might still be a better option. We could also do things like building the DAR and generating the typescript code in `genrules` to get better parallelism and caching.

Not going to attempt to do this in this PR.